proto/identity/associations/signature.proto

```diff
@@ -27,8 +27,6 @@ message SmartContractWalletSignature {
   // CAIP-10 string
   // https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-10.md
   string account_id = 1;
-  // Specify the block number to verify the signature against
-  uint64 block_number = 2;
```

Comment:
We still need the block number in here, since this is what we send to everyone else to verify the signature. We want them to verify it at a specific block

Reply:
Yeah, I realized that, was just about to move this back into draft.